pkg/okta/authentication_middleware.go

```diff
@@ -19,10 +19,14 @@ import (
 	"github.com/cmsgov/mint-app/pkg/userhelpers"
 )
 
+// These job codes define the environmentally specific potential codes which a user may possess
 const (
-	jobCodeUser       = "MINT_USER_NONPROD"
-	jobCodeAssessment = "MINT_ASSESSMENT_NONPROD"
-	jobCodeMAC        = "MINT MAC Users"
+	JobCodeTestUser       = "MINT_USER_NONPROD"
+	JobCodeTestAssessment = "MINT_ASSESSMENT_NONPROD"
+	JobCodeTestMACUser    = "MINT MAC Users"
+	JobCodeProdUser       = "MINT_USER"
+	JobCodeProdAssessment = "MINT_ASSESSMENT"
+	JobCodeProdMACUser    = "MINT_MAC_USER"
```

Comment:
I might need to confirm what this looks like across different environments. It's possible that this will be the same across prod/nonprod, but we should probably keep different const values for them just in case.

I should have some updates by tomorrow on what this value might look like, but have a feeling it'll also be `MINT MAC Users`

Reply:
Updated, ping me with the final codes when you have them and I'll update 👍